torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1549,8 +1549,16 @@ def error_inputs_hsplit(op_info, device, **kwargs):
                                   dtype=torch.float32,
                                   device=device),
                       args=(0,),)
+
+    # Incorrect type for indices_or_section argument
+    err_msg3 = ("received an invalid combination of arguments.")
+    si3 = SampleInput(make_tensor((S, S, S),
+                                  dtype=torch.float32,
+                                  device=device),
+                      args=("abc",),)
     return (ErrorInput(si1, error_regex=err_msg1),
-            ErrorInput(si2, error_regex=err_msg2),)
+            ErrorInput(si2, error_regex=err_msg2),
```

Comment:
I think we can just yield the ErrorInputs now, which might make this simpler and easier to read